src/Authentication/IAuthenticationService.cs

```diff
@@ -0,0 +1,7 @@
+using System.Threading.Tasks;
+
+namespace Microsoft.Graph.Cli.Authentication;
+
+interface IAuthenticationService {
```

Comment:
is the point of this interface to provide different implementations for the different authentication scenarios/flows? in which case I'd call it signinservice to clearly differentiate with the authentication providers. And I'd add a logout method as well.